R/scheduler.R

```diff
@@ -5,8 +5,12 @@ NULL
 scheduleTask <- function(func, delay) {
   force(func)
   later::later(function() {
-    op <- options(warn = 1)
-    on.exit(options(op))
+    # Make sure warn is at least 1 so that warnings are emitted immediately.
+    # (warn=2 is also OK, for use in debugging.)
+    if (getOption("warn") < 1) {
```

Comment:
Is it safe to assume this will always be a number, not NULL?

Reply:
R throws an error when you do `options(warn=NULL)` and `options(warn="foo")`, but it apparently does allow `NA` and `T`/`F`.  I'll add a check.

Reply:
Oh well in that case your original code is good enough for me, but if you feel like adding a check I don’t have an issue with that either.